docs/source/index.rst

```diff
@@ -16,6 +16,7 @@ Welcome to ESP Easy's documentation!
    Config/Config.rst
    Controller/_Controller.rst
    Hardware/Hardware.rst
+   Notify/_Notifications.rst
```

Comment:
IRL, the Notifications are next in line from the Devices, not next to the Hardware tab, so they should trade places with _plugin.rst. 🤓

Reply:
OK, will move it.